### ISSUE TITLE
Fix deploy (PyPI, conda-forge)

### ISSUE THREAD

**Issue by prisae** [prisae/fftlog#2]
There are various issues with the deploy and `setup.py`

- `setuptools_scm` with `setup` from `numpy (screws up version and MANIFEST and other things).
- Deploy from Travis fails at the moment (auth issue); also yields version 0.0.0.
- Manually uploaded to PyPi. However, because setuptools_scm fails, MANIFEST is exclude not include. So LICENSE file is missing on PyPi at the moment, very bad.
- Conda-forge recipe fails with compiling issues (and because LICENSE fail is missing).

So, in short, `fftlog` is not ready as package. (`pyfftlog` is all set.)

**Comment by prisae** [comment]
I think that was fixed in #3
